### ISSUE TITLE
Bug - Accessibility – All posts appear to contain images which lack alt text.

### ISSUE THREAD

**Issue by JamesONeill-DoF** [dof-dss/architecture-blog#31]
All blog posts currently contain images which lack alt text. E.g., https://blog.ea.digitalni.gov.uk/blog/product-launch/ea-team-launches-architecture-catalogue/

Alt text is required to ensure that the image purpose and meaning is conveyed to users of assistive technology (e.g., screen readers).

**Comment by JamesONeill-DoF** [comment]
I've re-tested this, and I'm not sure it's been resolved as yet. For each of the images I checked (the accessiblity post, the architecture catalogue post, and the AT/UT post), the alt text associated with each of the images was not descriptive of the image. 

For example: 

![image](https://user-images.githubusercontent.com/64086276/92619993-11852a00-f2ba-11ea-94b4-c98a577a3f07.png)


In this example, the screen-reader reads out "Graphic Current Article dot data dot image alt". This is not meaningful to the user.

**Comment by JamesONeill-DoF** [comment]
For reference, here's an image from one of the GDS blogs: 
https://accessibility.blog.gov.uk/2020/02/28/cleaning-our-way-to-accessibility/


![image](https://user-images.githubusercontent.com/64086276/92620782-f666ea00-f2ba-11ea-9a5c-258df704092e.png)
